kyuubi-server/src/test/scala/org/apache/kyuubi/operation/KyuubiOperationPerUserSuite.scala

```diff
@@ -37,6 +37,7 @@ class KyuubiOperationPerUserSuite
 
   override protected val conf: KyuubiConf = {
     KyuubiConf().set(KyuubiConf.ENGINE_SHARE_LEVEL, "user")
+    KyuubiConf().set(KyuubiConf.SERVER_INFO_PROVIDER, "SERVER")
```

Comment:
we should change the failed assertion instead of pining SERVER provider

Reply:
yes

Reply:
let's change it to
```
withSessionConf(Map(KyuubiConf.SERVER_INFO_PROVIDER.key -> "SERVER"))()() {
  withJdbcStatement() { statement => 
    ...
  }
}
```

Reply:
FYI:
```
    withSessionConf()(Map("key" -> "value"))(Map.empty) {
      withJdbcStatement() { _ =>
      }
    }
```